From a6de3c76db22db464dde40e5d753b4196527e7c8 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Wed, 19 Aug 2015 00:48:42 -0400
Subject: [PATCH] FileIOThread: Work Faster

If you have work, do your damn work as fast as you can.
---
 src/main/java/net/minecraft/server/FileIOThread.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main/java/net/minecraft/server/FileIOThread.java b/src/main/java/net/minecraft/server/FileIOThread.java
index 198b00f..9e17823 100644
--- a/src/main/java/net/minecraft/server/FileIOThread.java
+++ b/src/main/java/net/minecraft/server/FileIOThread.java
@@ -39,11 +39,11 @@ public class FileIOThread implements Runnable {
                 ++this.d;
             }
 
-            try {
+            /*try { // EMC
                 Thread.sleep(this.e ? 0L : 10L);
             } catch (InterruptedException interruptedexception) {
                 interruptedexception.printStackTrace();
-            }
+            }*/ // EMC
         }
 
         if (this.b.isEmpty()) {
-- 
1.9.1

